source/topics/privacy/dns-over-quic.rst

```diff
@@ -0,0 +1,142 @@
+:ref:`tls-service-pem:<unbound.conf.tls-service-pem>` .
+
+The resource consumption can be configured with ``quic-size: 8m``. More
+queries are turned away,
+:ref:`quic-size:<unbound.conf.quic-size>` .
+
+Libraries
+---------
+
+Unbound uses libngtcp2 for DNS over QUIC. This in turn requires a modified
+openssl library for quic support in the encryption for the quic transport.
+The modified openssl library is called openssl+quic. It is available for
+openssl versions 1.1.1 and 3.2.0, and so on.
+
+The modified openssl library is available from the openssl+quic repository, 
+`quictls <https://github.com/quictls/openssl>`__ . The libngtcp2 library
+is available from `ngtcp2 <https://github.com/ngtcp2/ngtcp2>`__ .
+
+The online documentation for libngtcp2 is available, `ngtcp2 <https://nghttp2.org/ngtcp2/>`__ . The ngtcp2-0.19.1 version tarball can be downloaded `ngtcp2-0.19.1 release <https://github.com/ngtcp2/ngtcp2/releases/tag/v0.19.1>`__ , instead
+of using the git checkout.
+
+For the openssl+quic also tarball downloads are available for releases,
+like for 3.0.10+quic, `openssl-3.0.10-quic1 release <https://github.com/quictls/openssl/releases/tag/openssl-3.0.10-quic1>`__ .
+
+For example unbound can be compiled with version ngtcp2-0.19.1, and with
+OpenSSL_1_1_1o+quic and openssl-3.0.10-quic1 .
+
+This is how to compile openssl+quic:
+
+.. code-block:: bash
+
+    git clone --depth 1 -b OpenSSL_1_1_1o+quic https://github.com/quictls/openssl openssl+quic
+    cd openssl+quic
+    ./config enable-tls1_3 no-shared threads --prefix=/path/to/openssl+quic_install
+    make
+    make install
+
+Fill in a good place to put the quic install. The example uses no-shared,
+so that the shared library search path later does not find the wrong dynamic
+library, but a shared library works too of course.
+
+The ngtcp2 library can be compiled like this:
+
+.. code-block:: bash
+
+    git clone --depth 1 -b v0.19.1 https://github.com/ngtcp2/ngtcp2 ngtcp2
+    cd ngtcp2
+    autoreconf -i
+    ./configure PKG_CONFIG_PATH=$PWD/../openssl+quic_install/lib/pkgconfig LDFLAGS="-Wl,-rpath,$PWD/../openssl+quic_install/lib" --prefix=/path/to/ngtcp2_install
+    make
+    make install
+
+Fill in the path to the openssl+quic install and path for where the libngtcp2
+install is created. The example sets the rpath to the directory to search for
+the dynamic library.
+
+The unbound server can be compiled with doq support, with the libngtcp2
+library, and the modified openssl library for quic support to libngtcp2, and
+this openssl library is then also used for TLS and other crypto calls, like
+for DNSSEC.
+
+Compile unbound then like this:
+
+.. code-block:: bash
+
+    ./configure <other flags> --with-ssl=/path/to/openssl+quic_install --with-libngtcp2=/path/to/ngtcp2_install LDFLAGS="-Wl,-rpath -Wl,/path/to/ngtcp2_install/lib" --prefix=/path/to/unbound_install
+    make
+
+Fill in the patch to the openssl+quic install and libngtcp2 install.
```

Comment:
```suggestion
Fill in the path to the openssl+quic install and libngtcp2 install.
```

Reply:
Applied the change to fix that mistake. Thank you for the comment.